.github/workflows/ci-release-gate.yml

```
@@ -38,180 +38,216 @@ jobs:
+    with:
+      image_repo: ${{ matrix.image_repo }}
+      digest_artifact_name: ${{ matrix.image_digest }}
+    secrets:
+      dockerhub_username: ${{ vars.DOCKER_HUB_USERNAME }}
+      dockerhub_token: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}    
+      
+     
+  dast-analysis:
+    name: DAST (Digest-based)
+    needs: [ build-push-frontend, build-push-backend, build-push-worker ]
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)